### ISSUE TITLE
Spawn-weight set to boolean value in ProjectZuluMobBiomeConfig.cfg

### ISSUE THREAD

**Issue by Taschenmogul** [soultek101/projectzulu1.7.10#13]
After updating to version 1.4a JustAnotherSpawner crashed on spawning a mob, stating that it could not cast an integer to a boolean.
After a short examination I found out that the first part of the spawn-weight entry for every mob in the "mob spawn biome controls" list was set to either "true" or "false", like in:

> S:"net.minecraft.world.biome.BiomeGenMutated.Cold Taiga M"=false:10:1:1
> S:"net.minecraft.world.biome.BiomeGenMutated.Desert M"=true:10:1:1

Peculiarly JustAnotherSpawner seems to have been handed the correct weights for all ProjectZulu mobs and the global spawn values remain untouched, still it crashes, as if ProjectZulu was expecting a boolean where JAS offered an integer.
OpenEye caught the exception so here is the crash page with crashlog and used versions:
http://openeye.openmods.info/crashes/ce6bf241cbddeed7d393c533a6044eed

**Comment by soultek101** [comment]
Greetings. Thank you for reporting this. There were no changes done to the format or data type of the PZ mobs since I took over so I am wondering what changes were done to JAS that would cause this. I do a thorough test of the scenario and provide feedback as soon as I can. Cheers!

**Comment by Taschenmogul** [comment]
Stumbling upon this after quite a few months and just wanted to make clear - the issue I reported does (and did) not exist, it was just my stupidity that made me think otherwise.
For some reason I just assumed that the format for entries was supposed to have an integer at the position where I saw booleans. Think I had just been fiddling too much with JustAnotherSpawner and got careless.
Actually the boolean values there are right where they belong and this issue should be closed.
